### PR TITLE
Number Validation & Apply form amountRequested modified

### PR DESCRIPTION
**Pull request by suzieahn1117**

### CODE REVIEW

**Review by jasonknee — CHANGES_REQUESTED** [review]
plz address comments

File: Poeteq.FundingAvenue/wwwroot/app/directives/ngCurrency.directive.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
+﻿(function () {
+    'use strict';
+
+    angular
+        .module('FundingAvenue')
+        .directive('ngcurrency', ngCurrency);
```
[the remark this answers is not in the document]
Comment (suzieahn1117) [reply Poeteq.FundingAvenue/wwwroot/app/directives/ngCurrency.directive.js]: ok

File: Poeteq.FundingAvenue/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -145,6 +145,8 @@
     <script src="~/app/controllers/contact.controller.js" asp-append-version="true"></script>
     <script src="~/app/controllers/apply.controller.js" asp-append-version="true"></script>
     <script src="~/app/directives/datepicker.directive.js" asp-append-version="true"></script>
+    <script src="~/app/directives/ngCurrency.directive.js" asp-append-version="true"></script>
+    <script src="~/app/providers/ngCurrencySettings.provider.js" asp-append-version="true"></script>
```
[the remark this answers is not in the document]
Comment (suzieahn1117) [reply Poeteq.FundingAvenue/Views/Shared/_Layout.cshtml]: forgot to remove the provider. thats an old one

File: Poeteq.FundingAvenue/wwwroot/app/controllers/apply.controller.js [hunk 3]
```diff
@@ -65,26 +65,43 @@
...
-                .then(function (response) {
-                    window.alert("Application Sent");
-                    console.log(response);
-                }, function (error) {
-                    console.log(error);
-                });
+            console.log($scope.applyInfo); 
+
+
+            //$http.post("Application/Form", $scope.applyInfo)
```
[the remark this answers is not in the document]
Comment (jasonknee) [reply Poeteq.FundingAvenue/wwwroot/app/controllers/apply.controller.js]: okay, np!

**Review by jasonknee — APPROVED** [review]
👍

**Pull request merged**